src/taglibs/migrate/all-tags/w-id.js

```diff
@@ -9,6 +9,9 @@ module.exports = function migrate(el, context) {
     );
 
     el.setAttributeValue("key", attr.value);
-    if (!el.hasAttribute("id")) el.setAttributeValue("id:scoped", attr.value);
+    const isHTML = el.tagDef && el.tagDef.html;
+    const isDynamic = /^\$\{.+\}$/.test(el.tagName);
```

Comment:
I think it would be better to check for `rawTagNameExpression` or `tagNameExpressions` instead of the RegExp like so `const isDynamic = Boolean(el.rawTagNameExpression)`.

Otherwise the PR LGTM.